static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -170,15 +176,29 @@ function RootCauseContent({
 function SuggestedFixSnippet({
   snippet,
   linesToHighlight,
+  repos,
 }: {
   linesToHighlight: number[];
+  repos: AutofixRepository[];
   snippet: AutofixRootCauseCodeContextSnippet;
 }) {
+  function getSourceLink() {
```

Comment:
This can be simplified by using `.find()`:

```ts
const repo = repos.find(repo => repo.name === snippet.repo_name && repo.provider === 'intergrations:github')
const sourceLink = `${repo.url}/blob/${repo.default_branch}/${snippet.file_path}`
```

Reply:
Done